blocks/shared-styles/_children/headings/index.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Heading from './heading';
+import HeadingSection from './section';
+
+jest.mock('fusion:context', () => ({
+  useFusionContext: jest.fn(() => ({
+    globalContent: {},
+    arcSite: 'the-sun',
+  })),
+}));
+
+jest.mock('fusion:themes', () => jest.fn(() => ({})));
+
+describe('Heading', () => {
+  it('should render as a h2', () => {
+    const wrapper = mount(<Heading />);
+
+    expect(wrapper.html()).toMatchInlineSnapshot(
+      '"<h1 class=\\"sc-bdVaJa bFuGRU\\"></h1>"',
+    );
+  });
+});
+
+describe('HeadingSection', () => {
+  it('should render as a h3', () => {
```

Comment:
same here? I'd expect this to render an h3 based on test description rather than an h2